apps/css-workshop/src/pages/menu.astro

```diff
@@ -38,239 +39,134 @@ import Icon_ from '../components/Icon.astro';
   <h2>Default</h2>
   <section id='demo-default'>
     <ul class='iui-menu' role='menu'>
-      <li
-        class='iui-list-item'
-        data-iui-actionable='true'
-        role='menuitem'
-        tabindex='0'
-        id='test-menu-1'
-      >
-        <span class='iui-list-item-content'>Option 1</span>
-      </li>
-      <li class='iui-list-item' data-iui-actionable='true' role='menuitem' tabindex='0'>
-        <span class='iui-list-item-content'>Option 2</span>
-      </li>
-      <li class='iui-list-item iui-disabled' data-iui-actionable='true'>
-        <span class='iui-list-item-content'>Option 3</span>
-      </li>
+      <Menu_ tabindex='0' id='test-menu-1' role='menuitem'>Option 1</Menu_>
+      <Menu_ tabindex='0' role='menuitem'>Option 2</Menu_>
+      <Menu_ disabled>Option 3</Menu_>
```

Comment:
All items should have `role='menuitem'`. Some of them are currently missing it.

Reply:
I have added `role=menuitem` to all items.